lib/git_hooks/installer.rb

```diff
@@ -1,33 +1,42 @@
 module GitHooks
   class Installer
-    attr_accessor :hook
-
     HOOK_SAMPLE_FILE = 'hook.sample'
 
-    def initialize(hook)
+    def initialize(hook, ruby_path = nil)
       @hook = hook
+      @ruby_path = ruby_path
     end
 
     def install(force = false)
-      return true if installed?
+      throw Exceptions::UnknownHookPresent.new(hook) if !force && installed?
+
+      hook_script = hook_template
```

Comment:
No problem.
The main point is not to `validate` the hooks based on its "symbolic-linkedness". 

Now I am checking the `installed?` like this

```ruby
  File.read(hook_path).match(/GitHooks.execute_pre_commits/)
```

I don't mind extracting these things to other classes, I just want to know what you think about not using the symbolic links anymore.